modules/ROOT/pages/proc_defining-and-creating-pipelineresources.adoc

```diff
@@ -10,53 +10,83 @@
 [discrete]
 === Procedure
 
-After you create `Task` s, create `PipelineResources` that contain the specifics of the Git repository and the image registry to be used in the `Pipeline` during execution as follows:
+After you create `Task`s, create `PipelineResources` that contain the specifics of the Git repository and the image registry to be used in the `Pipeline` during execution as follows:
 
-. Create a `PipelineResource` YAML file with a `PipelineResource` that defines the Git repository and reference for the PetClinic application as follows:
+. Create a `resources.yaml` file and inside of it create `PipelineResource`s which contain the specifics of the git repository and image registry to be used in the pipeline during execution.
+
+.. Create a `PipelineResource` that defines the git repository for the frontend application:
 +
 ----
 apiVersion: tekton.dev/v1alpha1
 kind: PipelineResource
 metadata:
-  name: petclinic-git
+  name: ui-repo
 spec:
   type: git
   params:
-  - name: url
-    value: https://github.com/spring-projects/spring-petclinic
+    - name: url
+      value: http://github.com/openshift-pipelines/vote-ui.git
 ----
 
-. Add another `PipelineResource`, which defines the OpenShift internal registry to which you want to push the PetClinic image, to the YAML file as shown below, and save it.
+.. Create a `PipelineResource` that defines the OpenShift internal image registry for the frontend image to be pushed to:
```

Comment:
The sentence constructions seems a bit awkward.
How about: Create a `PipelineResource` that defines the OpenShift internal image registry to which you want to push the frontend image.